.github/workflows/solidity.yml

```diff
@@ -167,3 +184,71 @@ jobs:
           hostname: ${{ secrets.GRAFANA_INTERNAL_HOST }}
           this-job-name: Prettier Formatting
         continue-on-error: true
+
+  publish-beta:
+    name: Publish Beta NPM
+    environment: publish
+    needs: [initialize, changes, lint, prettier, native-compile, prepublish-test]
+    runs-on: ubuntu-latest
+    if: needs.initialize.outputs.is-pre-release == 'true'
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Setup NodeJS
+        uses: ./.github/actions/setup-nodejs
+
+      - name: Publish to NPM
+        uses: smartcontractkit/.github/actions/ci-publish-npm@7b965cf1a0d58d05a9b8a45fffd929cd1ee18393 # feat/ci-publish-npm-0-3-0
+        with:
+          npm-token: ${{ secrets.NPM_TOKEN }}
+          create-github-release: false
+          # TODO: Remove --dry-run when ready to publish
```

Comment:
Should this be removed prior to merge?

Reply:
Two options:
1. We merge with `--dry-run`, then do a dry-run release. Then remove it in a separate PR
2. Remove it before merging, because I've tested it beforehand with `--dry-run`

Reply:
Either/or SGTM.